doc/appendix-de-kassensichv/receipt-case-definitions/receipt-case-definitions.md

```diff
@@ -11,14 +11,18 @@ The issuance and redemption of vouchers are processes with DSFinV-K requirements
 
 The difference is, that for single-purpose vouchers, the VAT rate is known already at transfer (issue) because it refers to a single purpose (service or product). On the other hand, in case of multi purpose vouchers the VAT rate is not known because it can be redeemed for different purposes (goods or services).
```

Comment:
Change to "multi-purpose voucher"

Add: A multi-purpos voucher is a money substitute.

Reply:
Done, thank you!